airbyte-webapp/src/components/CreateConnection/CreateConnectionNameField.tsx

```diff
@@ -13,20 +12,15 @@ export const CreateConnectionNameField = () => {
   const { formatMessage } = useIntl();
 
   return (
-    <Section>
+    <Section title={<FormattedMessage id="connection.title" />}>
```

Comment:
This needs to be reviewed with @Upmitt. The reason why this was set up this way in the first place was to match a design. However, I'm not sure where the latest design is located.